enum.c

```diff
@@ -2742,20 +2742,28 @@ each_with_index_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, memo))
 
 /*
  *  call-seq:
- *     enum.each_with_index(*args) { |obj, i| block } ->  enum
- *     enum.each_with_index(*args)                    ->  an_enumerator
+ *    each_with_index {|element, i| ..... } -> self
```

Comment:
`*args` should be added back, as it passes the args to `each`.  Similar to reverse_each and each_entry.

Reply:
Done.  I"ll study the examples thereof.